.viperlightignore

```diff
@@ -0,0 +1,27 @@
+# This file is used by Viperlight utility to override findings (false alarm)
+
+#NOTICE files
+workbench-core/datasets/NOTICE
+workbench-core/datasets-ui/NOTICE
+solutions/swb-app/NOTICE
+solutions/swb-reference/NOTICE
+solutions/swb-ui/NOTICE
+workbench-core/accounts/NOTICE
+workbench-core/accounts-ui/NOTICE
+workbench-core/authentication/NOTICE
+workbench-core/environments/NOTICE
+workbench-core/environments-ui/NOTICE
+workbench-core/example/infrastructure/NOTICE
+workbench-core/repo-scripts/repo-toolbox/NOTICE
+workbench-core/swb-common-ui/NOTICE
+workbench-core/infrastructure/NOTICE
+
+# Test files
```

Comment:
Can you add comments to these exclusions like `...test.ts:168 #foo@no-where.bar is not a real email`

This will make it easier later if the line number changes.

Reply:
@DaveMadsen updated with the changes you suggested.